README/ReleaseNotes/v624/index.md

```diff
@@ -44,6 +44,17 @@ The following people have contributed to this new version:
 
 ## Core Libraries
 
+Due to internal changes required to comply with the deprecation of Intel TBB's `task_scheduler_init` and related
+interfaces in recent TBB versions, as of v6.24 ROOT will not honor a maximum concurrency level set with
+`tbb::task_scheduler_init` but will require instead the usage of `tbb::global_control`:
+
+```cpp
+  //tbb::task_scheduler_init init(2); // does not affect the number of threads ROOT will use anymore
+
+  tbb::global_control c(tbb::global_control::max_allowed_parallelism, 2);
+  ROOT::TThreadExecutor p1;  // will use 2 threads
+  ROOT::TThreadExecutor p2(/*nThreads=*/8); // will still use 2 threads
+```
```

Comment:
```suggestion
`tbb::task_scheduler_init`. Instead, parallelism can be controlled in two ways:
- Set ROOT's parallelism level:
```cpp
  // Method 1:
  ROOT::EnableImplicitMT(2);
  ROOT::TThreadExecutor p1; // ROOT's parallelism level is initialised to 2
  
  // Method 2:
  ROOT::TThreadExecutor p2(8); // ROOT's parallelism level is initialised to 8.
```
The advantage of this method is that other TBB tasks can run concurrently with ROOT's tasks.
- Set TBB's global parallelism level using `tbb::global_control`:
```cpp
  //tbb::task_scheduler_init init(2); // does not affect the number of threads ROOT will use anymore

  tbb::global_control c(tbb::global_control::max_allowed_parallelism, 2);
  ROOT::TThreadExecutor p1;  // will use 2 threads
  ROOT::TThreadExecutor p2(/*nThreads=*/8); // will still use 2 threads
```
```

Reply:
Attention:
Github doesn't understand the suggestion, because ``` both delimit code and the suggestion ...

Reply:
isn't that more for documentation than for release notes? people look at release notes to see what changed, and the behavior of `ROOT::EnableImplicitMT` didn't change.

I could add a paragraphs in the docs somewhere (where?) about ROOT and parallelism and point to that from the release notes.

Reply:
Good idea. Maybe we ask during the team meeting?